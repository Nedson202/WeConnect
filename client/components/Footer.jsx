import React from 'react';
import '../index.scss'

/**
   * @description Creates footer.
   * 
   * @returns {object} JSX object
   * 
   * @memberof Footer
   */
const Footer = () => {
  return (
    <div>
      <footer>
        <div className="container">
          Â© 2018
          <a className="float-right" href="https://weconnect-api-service.herokuapp.com/api-docs">Api docs</a>
        </div>
      </footer>
    </div>
  );
}

export default Footer;
