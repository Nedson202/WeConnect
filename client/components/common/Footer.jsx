import React from 'react';

/**
 * @description Creates footer.
 * @returns {object} JSX object
 * @memberof Footer
 */
const Footer = () => (
  <div>
    <footer>
      <div className="container">
          Â© 2018
        <a className="float-right" href="https://weconnect-api-service.herokuapp.com/api-docs">Api docs</a>
      </div>
    </footer>
  </div>
);

export default Footer;
