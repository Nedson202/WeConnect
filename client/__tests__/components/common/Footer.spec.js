import React from 'react';
import expect from 'expect';
import { configure, shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Footer from '../../../components/common/Footer';

configure({ adapter: new Adapter() });

describe('Component: Footer', () => {
  it('should successfully render', () => {
    const wrapper = shallow(<Footer />);
    expect(toJSON(wrapper)).toMatchSnapshot();
    expect(wrapper.find('div').length).toBe(2);
    expect(wrapper.find('footer').length).toBe(1);
  });
});