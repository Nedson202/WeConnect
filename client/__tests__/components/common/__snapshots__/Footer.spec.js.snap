// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Footer should successfully render 1`] = `
<div>
  <footer>
    <div
      className="container"
    >
      Â© 2018
      <a
        className="float-right"
        href="https://weconnect-api-service.herokuapp.com/api-docs"
      >
        Api docs
      </a>
    </div>
  </footer>
</div>
`;
